extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -58,11 +59,14 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
+    /** @private {!../../../src/service/resources-impl.Resources} */
+    this.resources_ = null;
+
     /** @private {!boolean} */
     this.active_ = false;
 
     /** @private {!number} */
-    this.currentElementId_ = -1;
```

Comment:
nit: this is fine if you find it clearer, but I have noticed that usually AMP variable names err on the verbose side, and cosmetic renames tend to make PRs a little noisier.

Reply:
I renamed it to avoid exceeding the column count of 80 lines for certain lines of code that are of the form `this.verylongvariablename[this.anotherverylongvariablename].longpropertyname` to squelch lint. Sorry, couldn't justify splitting this into 2 lines...